tests/qadence/test_parameters.py

```diff
@@ -359,3 +359,8 @@ def test_sympy_modules() -> None:
     loaded_expr = deserialize(d)
     assert loaded_expr == expr
     assert evaluate(expr) == evaluate(loaded_expr)
+
+
+def test_numpy_paramtypes() -> None:
```

Comment:
What is this testing ?

Reply:
That the cases dont fail

Reply:
Right, what I meant is that will any error raise be picked up as a test failure ? Another point, if we do test that we should test all other numeric types allowed.

Reply:
yeah if the instantiation of the parameter instance fails, the test and hence the test suite fails. i think this case is just odd because python doesnt see `numpy.int64` as a child class of `int` which is the case for`numpy.float64` and `float`

Reply:
added tests for all numeric types. lets merge?